### ISSUE TITLE
Movement annoying, and suggestions for menus

### ISSUE THREAD

**Issue by nealol** [Sigton/bean_rpg#5]
Movement is annoying because there is no diagonal support, and if I hold W down for example, and press D, it will switch to going right. When I stop pressing D and am still pressing W it will not go forward.

Even just fixing this so that in the example I would keep going forward after I lifted my finger from D would be great, though diagonal would be better.

Menus:
Slight blur inside of menu?
Remove being able to move while inside a menu.
Clicking outside of the menu should close it.
Keyboard shortcuts for menus would be much nicer.
Changeable controls for interact button and menu shortcuts.

I really like the shadows by the way, they work amazingly well.

**Comment by Sigton** [comment]
Yeah they're good suggestions. I probably won't get round to adding them for a while but maybe one day.

Sigton
